core/src/test/java/org/infinispan/distribution/rehash/SharedStoreInvalidationDuringRehashTest.java

```diff
@@ -48,10 +43,6 @@
    private static final int NUM_KEYS = 20;
    private static final String TEST_CACHE_NAME = "testCache";
 
-   private final ConcurrentMap<Integer, ConcurrentMap<Object, AtomicInteger>> invalidationCounts = CollectionFactory.makeConcurrentMap();
-   private final ConcurrentMap<Integer, ConcurrentMap<Object, AtomicInteger>> l1InvalidationCounts = CollectionFactory.makeConcurrentMap();
-   private Map<Object, Integer> previousOwners = Collections.emptyMap();
```

Comment:
Does this still test anything? :)

Reply:
You would know better than me :D Should I remove? Or is there a recommendation as to how I could keep it?